vizro-core/examples/visual-vocabulary/pages/correlation.py

```diff
@@ -30,7 +30,17 @@
         """
         ),
         vm.Graph(figure=px.scatter(iris, x="sepal_width", y="sepal_length", color="species")),
-        make_code_clipboard_from_py_file("scatter.py"),
+        vm.Tabs(
+            tabs=[
+                vm.Container(
+                    title="Vizro dashboard", components=[make_code_clipboard_from_py_file("scatter.py", mode="vizro")]
+                ),
+                vm.Container(
+                    title="Plotly figure",
+                    components=[make_code_clipboard_from_py_file("scatter.py", mode="plotly")],
+                ),
+            ]
+        ),
```

Comment:
Maybe a util function can decrease code duplication here and in similar examples.